cfme/configure/access_control.py

```diff
@@ -1495,6 +1483,40 @@ def step(self):
 ####################################################################################################
 # RBAC PROJECT METHODS
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+class ProjectCollection(TenantCollection):
+    """Collection class for Projects under Tenants"""
+
+    def instantiate(self, name, description=None, parent=None, default=False):
+        return Project(collection=self, name=name, description=description, parent_tenant=parent)
+
+    def get_root_tenant(self):
+        # returning Tenant directly because 'My Company' needs to be treated like Tenant object,
+        # to be able to make child tenant/project under it
+        return Tenant(collection=self, name='My Company', _default=True)
+
+    def create(self, name, description, parent):
+        project = self.instantiate(name, description, parent)
+
+        if isinstance(parent, Project):
+            raise ValueError("Projects cannot have children.")
+
+        view = navigate_to(project.parent_tenant, 'Details')
+        view.toolbar.configuration.item_select('Add Project to this Tenant')
+
+        view = self.create_view(AddTenantView)
+        changed = view.form.fill({'name': name,
+                   'description': description})
+        if changed:
+            view.form.add_button.click()
+        else:
+            view.form.cancel_button.click()
+
+        view = self.create_view(ParentDetailsTenantView)
+        view.flash.assert_success_message('Project "{}" was saved'.format(name))
+
```

Comment:
**QUESTION**: You should probably also assert the view displayed here. Remember you can pass the parent tenant object to the create_view as well, so `is_displayed` works as expected.

Reply:
Again, is it necessary to explicitly check `is_displayed` after asserting flash message? @mfalesni